api/src/main/java/org/bahmni/module/events/api/configuration/JMSEventPublisherConfiguration.java

```diff
@@ -14,9 +12,9 @@
 
 import javax.jms.ConnectionFactory;
 
-@Conditional(EventPublishingToggleCondition.class)
+@Conditional(JMSEventPublishingToggleCondition.class)
```

Comment:
could have been done without a class using

> @ConditionalOnExpression

Reply:
Agree angshu but looks like @ConditionalOnExpression annotation is not present

Reply:
It comes with spring-boot-autoconfigure library which is not present